cmd/common/utils/parser-tableformatter.go

```diff
@@ -97,3 +103,27 @@ func (p *GadgetParser[T]) BuildColumnsHeader() string {
 func (p *GadgetParser[T]) TransformToColumns(entry *T) string {
 	return p.formatter.FormatEntry(entry)
 }
+
+func (p *GadgetParser[T]) TransformToTable(entries []*T) string {
+	maxWidth := 0
+	if term.IsTerminal(int(os.Stdout.Fd())) {
+		terminalWidth, _, err := term.GetSize(0)
+		if err == nil {
+			maxWidth = terminalWidth
+		}
+	}
```

Comment:
As we discussed, it would be great if we find a way to integrate this into the library. As you suggest, a helper like `GetTerminalWidth` could be enough for now.

Reply:
Solve using https://github.com/kinvolk/inspektor-gadget/pull/993.